CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.0] - 2020-10-14
+### Added
+* Support for device pairing to prevent shoulder surfing (i.e. make it impossible for someone in close physical proximity to a user to scan the QR code that was meant for the user)
+  * Introduced new frontend endpoints to manage device pairing
+  * Introduced `irmaclient` protocol version 2.7 including the necessary protocol changes
+  * The API of the `requestorserver` package has two new functions `SetFrontendOptions` and `PairingCompleted`
+  * A new server status `"PAIRING"` is introduced
+* A new function `SessionStatus` is available in the API of the `requestorserver` to get a channel with status updates of an IRMA session
+
+### Changes
+* The `irma.SessionPackage` struct now contains an extra field `FrontendAuth`
+* The `StartSession` function from the API of the `requestorserver` package now returns three values: the session pointer (type *irma.QR), the requestor token (type irma.RequestorToken) and the frontend authorization token (type irma.FrontendAuthorization)
+* The `token` parameter, as used by most functions in the API of the `requestorserver` package, now has the type `irma.RequestorToken`
+* The `server.Status` type has been moved to `irma.ServerStatus`; the related constants are also moved, e.g. from `server.StatusInitialized` to `irma.ServerStatusInitialized`
+
```

Comment:
Note to self: This API changes concern the `irmaserver` library and not the `requestorserver` library, update this.